### ISSUE TITLE
[2.2.0 RC 1] Metrics created in CLI not respecting random UID assignment

### ISSUE THREAD

**Issue by rclaire** [OpenTSDB/opentsdb#574]
With 2.2's random UID assignment enabled, metrics assigned with CLI tools do not have random UID assignment.

For example, **mkmetric** returning sequential UID's
```
/usr/share/opentsdb/bin/tsdb mkmetric test.metric.foo.d
metrics test.metric.foo.d: [0, 0, 5]
 /usr/share/opentsdb/bin/tsdb mkmetric test.metric.foo.e
metrics test.metric.foo.e: [0, 0, 6]
/usr/share/opentsdb/bin/tsdb mkmetric test.metric.foo.f
metrics test.metric.foo.f: [0, 0, 7]
```

However, metrics created automatically when submitted through telnet and with tsd.core.auto_create_metrics enabled do have proper random UID assignment

**Comment by manolama** [comment]
Fixed in d24683df68de1f65120c22f1a84afe877d474072. Thanks!

**Comment by bobrik** [comment]
@manolama I'm running `2.2.0RC3` and still see the issue:

```
# curl -s http://opentsdb/api/config | jq '.["tsd.core.uid.random_metrics"]'
"true"

# for i in `seq 1 10`; do curl -s http://opentsdb/api/uid/assign?tagv=woot$i | jq .; done
{
  "tagv": {
    "woot1": "00BFD7"
  }
}
{
  "tagv": {
    "woot2": "00BFD8"
  }
}
{
  "tagv": {
    "woot3": "00BFDB"
  }
}
{
  "tagv": {
    "woot4": "00BFDC"
  }
}
{
  "tagv": {
    "woot5": "00BFDD"
  }
}
{
  "tagv": {
    "woot6": "00BFDE"
  }
}
{
  "tagv": {
    "woot7": "00BFE0"
  }
}
{
  "tagv": {
    "woot8": "00BFE1"
  }
}
{
  "tagv": {
    "woot9": "00BFE2"
  }
}
{
  "tagv": {
    "woot10": "00BFE4"
  }
}
```
